compute/cker/src/DepthwiseConv.test.cc

```diff
@@ -84,6 +85,20 @@ template <typename T> class DepthwiseConvVerifier
                                   _filter_buffers.data(), output_shape, output.data()));
   }
 
+  void checkDeath(const nnfw::cker::DepthwiseConvParams &params,
+                  const nnfw::cker::Shape &input_shape, const T *input_data,
+                  const nnfw::cker::Shape &filter_shape, const T *filter_data,
+                  const nnfw::cker::Shape &bias_shape, const T *bias_data,
+                  const nnfw::cker::Shape &output_shape, const T *expected)
+  {
+    std::vector<T> output(output_shape.FlatSize());
+    EXPECT_DEATH(nnfw::cker::DepthwiseConvOp(params, input_shape, input_data, filter_shape,
```

Comment:
Binaries built in release mode do not execute assertions. You need to ensure this line is executed only on binaries built in debug mode.

Reply:
`EXPECT_DEBUG_DEATH` executes statement if it is RELEASE mode. It makes an systeam halt because the bias_data is null. How about wrapping it in define so that it only runs in debug mode?

Reply:
> EXPECT_DEBUG_DEATH executes statement if it is RELEASE mode. 

You are right. `EXPECT_DEBUG_DEATH` executes statement if it is RELEASE mode. 

> It makes an systeam halt because the bias_data is null.

I understand this now. If `bias_data` is `nullptr`, system is halted at another location, not `assert` statement in release mode.

> How about wrapping it in define so that it only runs in debug mode?

As we talked about offline, it would be better to allow optional bias and change that test to verify `DepthwiseConv2D` kernel with optional bias.